app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -259,6 +259,25 @@
           </p>
         </label>
 
+        <div class="medium-9 settings-item settings-item">
+          <label>
+            {{ $t('INBOX_MGMT.HELP_CENTER.LABEL') }}
+          </label>
+          <multiselect
+            v-model="selectedPortal"
+            track-by="id"
+            label="name"
+            :placeholder="$t('INBOX_MGMT.HELP_CENTER.PLACEHOLDER')"
+            selected-label
+            :select-label="$t('INBOX_MGMT.HELP_CENTER.SELECT_PLACEHOLDER')"
+            :deselect-label="$t('INBOX_MGMT.HELP_CENTER.REMOVE')"
+            :max-height="160"
+            :options="portal"
+            :allow-empty="true"
+            :option-height="104"
+          />
+        </div>
+
```

Comment:
I don't think we need multiselect for this, we can just use a single select right?